### PR TITLE
feat(pokemonBattlresList): add a percent after title

### LINKED ISSUE #218: Add a way to know the sum of the encounter percentages of wild Pokémon in a group
As a User
I would like to know the sum of the encounter percentages of wild Pokémon in a group
So that I don't have to manually calculate it

### PR DESCRIPTION
**Pull request by Menthx**
Thank you for your contribution to the **Pokémon Studio** repo.

Before submitting this PR into the develop branch, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You are following the [Code guidelines](../CodeGuidelines.md)
- [x] You tested your code to make sure it does what it is supposed to do

## Description

I would like to know the sum of the encounter percentages of wild Pokémon in a group
So that I don't have to manually calculate it

## Tests to perform
When no pokemon is present, tag should not be displayed
When some creatures are present, make sure tag is displayed

https://github.com/PokemonWorkshop/PokemonStudio/issues/218

### CODE REVIEW

**Review by AerunDev — APPROVED** [review]
Simple et efficace, bien joué à toi ! 😄

File: src/views/components/pokemonBattler/PokemonBattlerList.tsx [hunk 1]
```diff
@@ -99,11 +100,16 @@ export const PokemonBattlerList = ({ title, encounters, disabledImport, from }:
...
+  const totalEncounterChance = encounters.map((encounter) => encounter.randomEncounterChance).reduce((a, b) => a + b, 0);
+
   return (
     <PokemonBattlerListComponent size="full" data-noactive>
       <PokemonBattlerListHeader>
-        <div className="title">{title}</div>
-        <div className="buttons">
+        <div className="header">
+          <div className="title">{title}</div>
+          {!!totalEncounterChance && <Tag className="chance">{`${totalEncounterChance}%`}</Tag>}
```
Comment (NuriYuri) [inline src/views/components/pokemonBattler/PokemonBattlerList.tsx]: Comme le reduce commence à 0, il n'est pas nécessaire de tester l'existence de la valeur, ou alors tu peux écrire totalEncounterChance > 0

**Review by NuriYuri — APPROVED** [review]

File: src/views/components/pokemonBattler/PokemonBattlerList.tsx (hunk 1 above)
Comment (Menthx) [reply src/views/components/pokemonBattler/PokemonBattlerList.tsx]: Yes, rendre la condition plus précise !

**Review by Palbolsky — CHANGES_REQUESTED** [review]
Le pourcentage ne doit pas s'afficher pour les dresseurs : 

image

**Comment by Menthx** [comment]
merci Palb, je n'avais pas vu ce bout là !

**Review by Palbolsky — APPROVED** [review]
Parfait bien joué :)

**Review by NuriYuri — APPROVED** [review]

**Pull request merged**